image/cli/mascli/functions/pipeline_show_config

```diff
@@ -106,6 +106,13 @@ function pipeline_show_config() {
   echo_h4 "MongoDB" "    "
   echo_reset_dim "Install Namespace ......... ${COLOR_MAGENTA}${MONGODB_NAMESPACE:-<default>}"
 
+  reset_colors
```

Comment:
As above, collapse all this into a single "UDS settings" .. even if it's DRO .. it's the same thing, don't let the name change confuse users.

Reply:
done